src/electron/process_manager.ts

```diff
@@ -337,7 +334,6 @@ class SsLocal extends ChildProcessHelper {
     args.push('-p', '' + config.port);
     args.push('-k', config.password || '');
     args.push('-m', config.method || '');
-    args.push('-t', SSLOCAL_TIMEOUT_SECS.toString());
```

Comment:
Do we want this to be in a separate PR?  We don't want to have to revert the library upgrade if this turns out to need rolled back or whatever

Reply:
It's a small change, so I think it's okay. We can cherry-pick the individual commit (af5e14c) if we need to.